frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -278,11 +271,14 @@ class StructuredQuery extends AtomicQuery {
   setDefaultQuery(): StructuredQuery {
     const table = this.table();
 
+    const query = this.getMLv2Query();
+    const isEmpty = Lib.databaseID(query) == null;
```

Comment:
This is just an intermediary step until I remove `setDefaultQuery` in a separate PR.

See: https://github.com/metabase/metabase/issues/37506